package.yaml

```diff
@@ -36,11 +36,11 @@ dependencies:
 - aeson
 - time
 - warp
-- monad-logger
 - ekg
 - ekg-core
 - universum
 - unordered-containers
+- katip
```

Comment:
Not in alphabetic order..

Reply:
Yeah, there was some PR which sorts alphabetically. Now it's merged so I can make this list better. Thanks for pointing that 👍